### PR TITLE
Add config option to make images square

### PR DESCRIPTION
**Pull request by art0007i**
This option will be false by default
Since we have the mod loader config system I think a toggle for the space wasting square image mode could be added to the official mod

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 
 A fork of this mod is also available [here](https://github.com/art0007i/GifImporter).<br>
 This fork makes the generated sprite sheets into a almost square resolution, at the cost of some wasted space in certain cases.
 
 ## Installation
 1. Install [NeosModLoader](https://github.com/zkxs/NeosModLoader).
-1. Place [GifImporter.dll](https://github.com/kawaiiamber/GifImporter/releases/latest/download/GifImporter.dll) into your `nml_mods` folder. This folder should be at `C:\Program Files (x86)\Steam\steamapps\common\NeosVR\nml_mods` for a default install. You can create it if it's missing, or if you launch the game once with NeosModLoader installed it will create the folder for you.
+1. Place [GifImporter.dll](https://github.com/art0007i/GifImporter/releases/latest/download/GifImporter.dll) into your `nml_mods` folder. This folder should be at `C:\Program Files (x86)\Steam\steamapps\common\NeosVR\nml_mods` for a default install. You can create it if it's missing, or if you launch the game once with NeosModLoader installed it will create the folder for you.
```
Comment (zkxs) [inline README.md:12]: These readme changes look unintentional

File: README.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
 # GifImporter
 
+This is my fork of the gif importer, it tries to fit the sprite sheets into a square resolution, but it will sometimes create redundant empty frames at the end of the gif.
```
Comment (zkxs) [inline README.md:3]: Again, readme mistake?

Comment (art0007i) [reply README.md]: True, I overlooked that

**Comment by kawaiiamber** [comment]
Could you squash your commits? Ideally one commit per file in this case.

**Comment by art0007i** [comment]
I have no idea how to squash commits

**Comment by kawaiiamber** [comment]
> I have no idea how to squash commits

Since there's 13 commits -

```
git rebase -i HEAD~13
```
Then choose `squash` on all but the first one, choose `pick`. Then: `git push -f origin master`

**Comment by art0007i** [comment]
Ok I think I've done it

**Pull request merged**